### ISSUE TITLE
Provide imperative configuration through CLI-1

### ISSUE THREAD

**Issue by ashwathishiva** [qlik-oss/sense-installer#72]
Allow usage of CLI to create new qliksense application configuration instances and set properties. Should be somewhat aligned with the way kubectl config works. 

```shell
> kubectl qliksense config set-context myqliksense
> kubectl qliksense config set profile=docker-desktop
> kubectl qliksense config set namespace=qliksense
> kubectl qliksense config set storageClassName=efs
> kubectl qliksense config set rotateKeys=yes
> kubectl qliksense config set git.repository="https://github.com/my-org/qliksense-k8s"  # Uses kubernetes operator when specified
> kubectl qliksense config set-configs qliksense[name=acceptEULA]="yes"

> kubectl qliksense config set profile=docker-desktop namespace=qliksense storageClassName=qliksense rotateKeys=yes
```
```yaml
apiVersion: qlik.com/v1
kind: QlikSense
metadata:
  name: myqliksense
spec:
  git:
    repository: https://github.com/my-org/qliksense-k8s
  profile: manifests/docker-desktop
  namespace: myqliksense
  storageClassName: efs
  rotateKeys: "no"
  releaseName: myqliksense
  configs:
  - qliksense:
    - name: acceptEULA
      value: "yes"
  secrets:
  - qliksense:
    - name: mongoDbUri
      value: "mongo://mongo:3307"
```

CLI should store configuration as: `~/.qliksense/contexts/myqliksense/myqliksense.yaml`
```yaml
apiVersion: qlik.com/v1  
kind: QlikSense
metadata:
  name: myqliksense
  labels:
    version: v1.0.0
spec:
  profile: manifests/docker-desktop
  namespace: myqliksense
  storageClassName: efs
  configs:
  - qliksense:
    - name: acceptEULA
      value: "yes"
  secrets:
  - qliksense:
    - name: mongoDbUri
      value: "mongo://mongo:3307"

~/.qliksense/contexts/myotherqliksense/myotherqliksense.yaml

```yaml
apiVersion: qlik.com/v1
kind: QlikSense
metadata:
  name: myotherqliksense
  labels:
    version: v2.0.0
spec:
  gitRepo: https://github.com/myorg/qliksense-k8s
  profile: manifests/docker-desktop
  namespace: myotherqliksense
  storageClassName: efs
  rotateKeys: "yes"
  releaseName: myotherqliksense
  configs:
  - qliksense:
    - name: acceptEULA
      value: "yes"
  secrets:
  - qliksense:
    - name: mongoDbUri
      value: "mongo://atlasmongo:3307"
```
All context info is stored in: `~/.qliksense/config.yaml`:
```yaml
apiVersion: config.qlik.com/v1
kind: QliksenseConfig
metadata:
  name: QliksenseConfigMetadata
spec:
  contexts:
  - name: qlik-default
    crFile: /Users/fki/.qliksense/contexts/qlik-default/qlik-default.yaml
  - name: myqliksense
    crFile: /Users/fki/.qliksense/contexts/myqliksense/myqliksense.yaml
  currentContext: qliksense-default
```

**Comment by ashwathishiva** [comment]
Here are the PRs that address the above requirement: 
https://github.com/qlik-oss/sense-installer/pull/66
https://github.com/qlik-oss/sense-installer/pull/69
https://github.com/qlik-oss/sense-installer/pull/70
https://github.com/qlik-oss/sense-installer/pull/75
https://github.com/qlik-oss/sense-installer/pull/78
https://github.com/qlik-oss/sense-installer/pull/81

**Comment by ashwathishiva** [comment]
At this point, we are able to use CLI to create new application configuration and set properties, just like we would use `kubectl`. 

We are able to create a default qliksense context, when no context is specified, create new contexts, configure different properties(namespace, profile, storage class, rotate keys and git repository) into the context. We are also able to set properties as key-value pairs using the `set-config` command. 

Supported commands are: 
`> qliksense config set-context myqliksense
> qliksense config set profile=docker-desktop
> qliksense config set namespace=qliksense
> qliksense config set storageClassName=efs
> qliksense config set rotateKeys=yes
> qliksense config set git.repository="https://github.com/my-org/qliksense-k8s"  # Uses kubernetes operator when specified
> qliksense config set-configs qliksense[name=acceptEULA]="yes"

> kubectl qliksense config set profile=docker-desktop namespace=qliksense storageClassName=qliksense rotateKeys=yes`

We need to support addition of secrets using the CLI as well, this work is addressed as part of #32.
